charts/posthog/tests/events-deployment.yaml

```diff
@@ -183,6 +183,32 @@ tests:
             name: SENTRY_DSN
             value: sentry.endpoint
 
+  - it: sets env vars from global, web and events sections
```

Comment:
Can we add a test with both `web.env` and `events.env`? I think with the current logic we will hit a key collision if we specify the same key in both places (I might be wrong tho!)